### PR TITLE
Add fusion playbooks

### PR DESCRIPTION
**Pull request by vromash**
**SUMMARY**
Add fusion playbooks. Moved from [fusion-client-devkit](https://github.com/PureStorage-OpenConnect/fusion-client-devkit).

### CODE REVIEW

File: fusion/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Ansible Playbooks for the Fusion DevKit
+
+## Summary
+The goal of this document is to help you set up and understand the functionality and scope of the ansible playbooks here providers.
```
Comment (jankodera-ps) [inline fusion/README.md]: providers -> provided?

File: fusion/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+
+### How run a playbook
+```
+ansible-playbook playbook_name.yml
+```
+
+### Environment Variables
+In almost every playbook, there are 2 lines that required a enviroment variable
```
Comment (jankodera-ps) [inline fusion/README.md]: required -> require?

File: fusion/README.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+```
+In this case, the variables are: API_CLIENT and PRIV_KEY_FILE
+To set them, you can temporaly enable them with:
+```
+export API_CLIENT='pure1:apikey:123456789'
+export PRIV_KEY_FILE='/home/user/key.pem'
+```
+in the case of ```PRIV_KEY_FILE```, the path to the ```key.pem``` need to be aboslute.
+
+IF you prefer to not use enviroment variables, you can change the values inside the playbook:
```
Comment (jankodera-ps) [inline fusion/README.md]: IF -> If?

File: fusion/README.md (hunk 1 above)
Comment (vromash) [reply fusion/README.md]: I haven't checked README as I only moved it from on repo to another.
But I can go through and check any typos then

**Review by DenDen039 — APPROVED** [review]
LGTM. But I don't know appropriate structure for this repo. We need to wait for @sdodsley review

File: fusion/README.md (hunk 1 above)
Comment (vromash) [reply fusion/README.md]: Went through README and fixed all typos I could find

**Review by jankodera-ps — APPROVED** [review]

**Review by sdodsley — APPROVED** [review]
LGTM

**Pull request merged**